ansible/roles-infra/infra-ibm-ssh-key/tasks/main.yml

```diff
@@ -13,11 +13,12 @@
 #     | cut -d " " -f2
 #   register: r_ssh_key_fingerprint
 
+# If we only need one key on the sandbox, we can use the task below to do both, check and create
+# Idempotency tested
```

Comment:
This will not work. It actually checks the fingerprint of the public key you are uploading, so if you have a key - no matter what it is named or what RG it is in - and try to upload another, it fails. This is why i have the previous task roughed out.

Reply:
It will fail if you use the same public_key content to create a new one but should work if you use a new public_key that does not hit with the previously created one.